File: FrontEndTest/src/models/FilterContactModel.js
export class FilterContactModel{

    constructor(emailAddress, phoneNumer){
        this.emailAddress = emailAddress
        this.phoneNumer = phoneNumer;
    }

    get EmailAddress() {
        return this.emailAddress
    }

    set EmailAddress(value) {
        this.emailAddress = value
    }

    get PhoneNumer() {
        return this.phoneNumer
    }

    set PhoneNumer(value) {
        this.phoneNumer = value
    }

}

File: BackEndTest/WebApiTest/WebApiTest/Models/FilterContactModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using WebApiTest.Utils;

namespace WebApiTest.Models
{
    public class FilterContactModel
    {
        public string EmailAddress { get; set; }

        public string PhoneNumer { get; set; }


        public bool IsEmpty => EmailAddress.IsNullOrEmpty() || PhoneNumer.IsNullOrEmpty();
    }
}
